css/styles.css

```diff
@@ -154,47 +256,49 @@ body {
   transition: transform 0.2s;
 }
 
-#myRecentWorksContainer {
-  display: grid;
-  grid-template-columns: auto;
+#anchor-button {
+  margin: 26px auto0;
+}
+
+/* My Recent Works Mobile Styles */
+
+#my-recent-works-container {
   grid-template-rows: repeat(7, auto);
   row-gap: 24px;
-  justify-content: center;
-  margin-bottom: 20px;
+  margin-top: 0;
+  padding-top: 5px;
 }
 
 #recentWorkSeparator {
   background-color: var(--button-enabled-color);
   border-radius: 5px;
   width: 48px;
   height: 4px;
-  margin: 25px auto 30px;
+  margin: 25px auto 38px;
   border: none;
 }
 
-.workCard {
+.work-card {
```

Comment:
:red_circle: Your project looks great :clap:. I think you can improve it. Please make sure all the  `.work-card` elements have the same width to match the Figma Design

**Your Design**
![card not distrubuted](https://user-images.githubusercontent.com/26736582/200073293-24506f45-4cbc-4fad-a13e-687378096df1.png)

**Figma Design**
![samewidth](https://user-images.githubusercontent.com/26736582/200073412-303dfd94-2368-4aa0-9de6-f05d3c378ec7.png)